vcpkg_overlays/antlr3/portfile.cmake

```diff
@@ -0,0 +1,35 @@
+vcpkg_download_distfile(LIB_C
+    URLS "https://github.com/antlr/website-antlr3/raw/refs/heads/gh-pages/download/C/libantlr3c-${VERSION}.tar.gz"
+    FILENAME "libantlr3c-${VERSION}.tar.gz"
+    SHA512 8edb243d745ff5bf3b15940f124d1255a9ca965cb656a73a558aed7fa07effcd7620f23dc692e5d5169a03200254836dd57af3ce444ba225281a5b721497e211
+)
+vcpkg_extract_source_archive(SOURCE_PATH
+    ARCHIVE "${LIB_C}"
+)
+vcpkg_configure_make(
+    SOURCE_PATH "${SOURCE_PATH}"
+    COPY_SOURCE
+    OPTIONS
+        --enable-64bit
```

Comment:
./configure will need prompting for certain architectures. linux-arm64 in particular needs to have this added as an option:

--build=aarch64-unknown-linux

I suspect macOS (arm64/aarch64) is in the same boat.